app/models/dialog_field_sorted_item.rb

```diff
@@ -96,6 +96,9 @@ def sort_data(data_to_sort)
   end
 
   def determine_selected_default_value
+    if dynamic? && force_multi_value && !default_value.kind_of?(Array)
```

Comment:
`force_multi_value?` in base model will return false and in special models it will return `what you need`